sympy/solvers/pde.py

```diff
@@ -897,11 +897,11 @@ def pde_separate_add(eq, fun, sep):
     """
     Helper function for searching additive separable solutions.
 
-    Consider an equation of two independent variables x, y and a dependent
-    variable w, we look for the product of two functions depending on different
+    Consider an equation of two independent variables x and t and a dependent
```

Comment:
Ah, but as the 'Consider...' is only talking about an example, and the example has only two independent variables, the text suggested by @smichr is fine.  I've adopted it and pushed it to my fork of sympy, branch gdmcbain:pde_separate-doc.